src/app/components/school-menu/school-list.tsx

```diff
@@ -12,10 +12,15 @@ interface SchoolListProps {
   color: 'dark' | 'light';
 }
 
-export const SchoolList = ({ list, color }: SchoolListProps) => (
-  <ul className="school-list">
-    {list?.map((item, index) => (
-      <SchoolItem item={item} index={index} key={index} color={color} />
-    ))}
-  </ul>
-);
+export const SchoolList = ({ list, color }: SchoolListProps) => {
+  const className = (!!list && !!list[0] && 'description' in list[0]) 
```

Comment:
It seems overloaded. Will such a code work? list[0].description ? 'school-list school-list_description' : 'school-list'

Reply:
The option you suggested will not work.
In my mind, the best way to avoid this is to divide this component into two components (a list of courses and a list of page navigation. Or rework the `SchoolItem` component.
In any case, it comes out of scope of this fix.
I think it's worth creating a task to refactor the `school-menu` component and redo it completely.